### ISSUE TITLE
interval=1820 gives an error

### ISSUE THREAD

**Issue by rajil** [chenxiaolong/ipmi-fan-control#39]
I want to give a large interval (more than 30 minutes) to the HDD zone so it can play nice with hd-idle. Unfortunately, it bombs out

```
ERROR ipmi_fan_control] Failed to parse config "/etc/ipmi-fan-control.toml": invalid value: integer `1820`, expected u8 for key `zones.interval` at line 73 column 12
```

**Comment by chenxiaolong** [comment]
Thanks, I'll get this fixed really quickly. It's because I'm using an 8 bit integer to represent the interval. I'll change it to a larger type.

**Comment by chenxiaolong** [comment]
This should be fixed in version 0.3.6. If you're using the prebuilt packages, those should be updated shortly.
